src/components/browseButton/index.js

```diff
@@ -87,6 +90,7 @@ export function setup ({
 
             // Display a PDF and annotations.
             restoreBeforeState(current)
+            _restoreBeforeStatus(current)
```

Comment:
Can you change it to optional like below?
```
_restoreBeforeStatus && _restoreBeforeStatus(current)
```
PDFAnno doesn't need this, so optional paramer is better for me.